src/zenml/integrations/seldon/model_deployers/seldon_model_deployer.py

```diff
@@ -112,7 +108,7 @@ def get_active_model_deployer() -> "SeldonModelDeployer":
                 "Core. You can create a new stack with a Seldon Core model "
                 "deployer component or update your existing stack to add this "
                 "component, e.g.:\n\n"
-                "  'zenml model-deployer register seldon --type=seldon "
+                "  'zenml model-deployer register seldon --flavor=seldon "
```

Comment:
Similar to above, could be replaced with the flavor constant

Reply:
Thanks! Changed it.